shared/django_apps/codecov_auth/tests/factories.py

```diff
@@ -13,7 +13,7 @@
     Session,
     TokenTypeChoices,
     User,
-    UserToken,
+    UserToken, Account, AccountsUsers, OktaSettings, StripeBilling, InvoiceBilling,
```

Comment:
Hmm.... it's ignoring formatting this file because we excluded it in https://github.com/codecov/shared/blob/aa771d470f05f5bbffcd8ac57ef047369914e197/ruff.toml#L29 . I believe it was added because cleaning up tests was a pain for linter, but we should probably add it back at some point. @nora-codecov Would you mind manually running `ruff format ./shared/django_apps/codecov_auth/tests/factories.py ` on the test files? They won't run unless you specify the full filepath. cc/ @aj-codecov

Reply:
@michelletran-codecov We can definitely add it back! There will be some rules we'll have to turn off again but I'm good with that 👌 It's something I regret doing tbh...

Reply:
Btw @aj-codecov is our PM :)

Reply:
🤦 Whoops!